vizro-ai/.readthedocs.yaml

```diff
@@ -11,6 +11,8 @@ build:
     python: "3.11"
   commands:
     - pip install hatch
-    - cd vizro-ai/ && hatch run docs:build && hatch run docs:link-check
+    - cd vizro-ai/ && hatch run docs:pip freeze
+    - cd vizro-ai/ && hatch run docs:build
+    - cd vizro-ai/ && hatch run docs:link-check
```

Comment:
Why do we need `hr docs:pip freeze` (where can we find this output)?

Reply:
It's in logs for the build on RTD. e.g. https://app.readthedocs.org/projects/vizro/builds/24744550/
![image](https://github.com/mckinsey/vizro/assets/49395058/e6e72487-3ddf-44ef-b96a-f6a2c2c4d6fe)